.github/workflows/ci.yml

```diff
@@ -98,6 +98,10 @@ jobs:
     upload_pypi:
         needs: [pass]
         runs-on: ubuntu-latest
+        environment: pypi
```

Comment:
Apparently not strictly required. But on pypi side we can enforce that the 'environment' string matches this